modules/thermo-cycler/thermo-cycler-arduino/thermo-cycler-arduino.ino

```diff
@@ -527,9 +683,56 @@ void read_gcode()
         case Gcode::deactivate_all:
           deactivate_all();
           break;
+        case Gcode::deactivate_plate:
+          deactivate_plate();
+          break;
         case Gcode::get_device_info:
           gcode.device_info_response(device_serial, device_model, FW_VERSION);
           break;
+        case Gcode::set_offset_constants:
+          /**** Save offsets to EEPROM & update the current constants *****/
+          if (gcode.pop_arg('A'))
+          {
+            if (!eeprom.set_offset_constant(OffsetConst::A, gcode.popped_arg()))
+            {
+              gcode.response("EEPROM ERROR","offset A not saved");
+            }
+          }
+          if (gcode.pop_arg('B'))
+          {
+            if (!eeprom.set_offset_constant(OffsetConst::B, gcode.popped_arg()))
+            {
+              gcode.response("EEPROM ERROR","offset B not saved");
+            }
+          }
+          if (gcode.pop_arg('C'))
+          {
+            if (!eeprom.set_offset_constant(OffsetConst::C, gcode.popped_arg()))
+            {
+              gcode.response("EEPROM ERROR","offset C not saved");
+            }
+          }
+          update_offset_constants();
+          break;
+        case Gcode::get_offset_constants:
+          /**** Get current constants ****/
+          gcode.response("a", String(const_a, 4));
```

Comment:
looks like we require upper case in the request and then provide lower case here, we should keep it consisent